Dockerfile

```diff
@@ -17,10 +17,11 @@ COPY show-pom-version.rb /jenkins/src/show-pom-version.rb
 WORKDIR /jenkins/src/
 RUN mvn clean install --batch-mode -Plight-test
 
-# The image is based on the previous weekly, new changes in jenkinci/docker are not applied
-FROM jenkins/jenkins:latest
+# The image is based on https://github.com/jenkinsci/docker/tree/java11
+# All documentation is applicable
+FROM jenkins/jenkins-experimental:2.138.1-jdk11
```

Comment:
Should be replaced by the common release once https://github.com/jenkinsci/docker/pull/762 from @batmat lands in the weekly release

Reply:
@oleg-nenashev `docker pull jenkins/jenkins:2.152-jdk11` now is released.